README.md

```diff
@@ -18,31 +18,51 @@ The SQL schema for ClickHouse must be created on each ClickHouse node and looks
 ```sql
 CREATE DATABASE IF NOT EXISTS logs ENGINE=Atomic;
 
-CREATE TABLE IF NOT EXISTS logs.logs_local ON CLUSTER '{cluster}' (
-  timestamp DateTime64(3) CODEC (Delta, ZSTD(1)),
-  cluster LowCardinality(String) CODEC (ZSTD(1)),
-  namespace LowCardinality(String) CODEC (ZSTD(1)),
-  app String CODEC (ZSTD(1)),
-  pod_name String CODEC (ZSTD(1)),
-  container_name String CODEC (ZSTD(1)),
-  host String CODEC (ZSTD(1)),
-  fields_string Nested(key String, value String) CODEC (ZSTD(1)),
-  fields_number Nested(key String, value Float64) CODEC (ZSTD(1)),
-  log String CODEC (ZSTD(1))
-) ENGINE = ReplicatedMergeTree()
-  TTL toDateTime(timestamp) + INTERVAL 30 DAY DELETE
-  PARTITION BY toDate(timestamp)
-  ORDER BY (cluster, namespace, app, pod_name, container_name, host, -toUnixTimestamp(timestamp));
+CREATE TABLE IF NOT EXISTS logs.logs_local ON CLUSTER `{cluster}`
+(
+    `timestamp` DateTime64(3) CODEC(Delta, LZ4),
+    `cluster` LowCardinality(String),
+    `namespace` LowCardinality(String),
+    `app` LowCardinality(String),
+    `pod_name` LowCardinality(String),
+    `container_name` LowCardinality(String),
+    `host` LowCardinality(String),
+    `fields_string.key` Array(LowCardinality(String)),
+    `fields_string.value` Array(String) CODEC(ZSTD(1)),
```

Comment:
As you wrote in the PR

> CODEC(ZSTD(1)) -> no codec or CODEC(LZ4)

should this also be changed for the `fields_string.value`

Reply:
> should this also be changed for the fields_string.value

It's needs to be tested :)  (Most likely by you, as you have good dataset)
Now, i can't say for sure.

It's fairly easy to do such test, here is article about it:
https://kb.altinity.com/altinity-kb-schema-design/codecs/altinity-kb-how-to-test-different-compression-codecs/

Reply:
Thanks for the link 🙂. I will merge your PR and try to compared based on our data.